source/core/src/main/com/deco2800/game/components/story/StoryBase.java

```diff
@@ -1,5 +1,30 @@
 package com.deco2800.game.components.story;
 
```

Comment:
might be worth adding short summary on why we need story base here as well. You could probably just add what you've mentioned in the pull request 'A story base object is what the manager calls upon to control the story. It also stores the entities necessary for the story'